webapp/src/js/enketo/widgets/db-object-widget.js

```diff
@@ -89,12 +91,20 @@ const changeHandler = function() {
     // https://github.com/enketo/enketo-core/issues/910
     // Re-validate the current question now that we have loaded the doc data.
     // This will clear any constraint errors that were resolved by the doc data.
-    window.CHTCore.Enketo.getCurrentForm().validateContent($this.parent());
+    const currentForm = getCurrentForm();
+    if (currentForm) {
```

Comment:
Would it actually make more sense to do this check _before_ the `updateFields` call? That way, I don't think we would need any extra logic down in `updateFields`...

```js
const currentForm = getCurrentForm();
if (!currentForm) {
    return;
}
updateFields(doc, keyRoot, index, field);
// https://github.com/enketo/enketo-core/issues/910
// Re-validate the current question now that we have loaded the doc data.
// This will clear any constraint errors that were resolved by the doc data.
currentForm.validateContent($this.parent());
```